docs/docs/pyproject.md

```diff
@@ -239,10 +239,23 @@ mandatory = "^1.0"
 # below `extras`. They can be opted into by apps.
 psycopg2 = { version = "^2.7", optional = true }
 mysqlclient = { version = "^1.3", optional = true }
+rich = { version = "^8.0", optional = true }
+emoji = { version = "^0.6", optional = true }
 
 [tool.poetry.extras]
 mysql = ["mysqlclient"]
 pgsql = ["psycopg2"]
+styles = ["rich", "emoji"]
+```
+
+`extras` can only be used with packages defined in the `dependencies` section.
+
+When pip installing the package, the dependencies specified in the `extras` section can be installed by providing
```

Comment:
We should really be talking about `pip` as an example of a PEP 517 front-end, not necesarily the defacto option that uses this. The conversation should also probably reference PEP 508 when mentioning how extras are consumed. Especially because "extras" is not something specific to poetry.

Reply:
Great point! I will update this